### ISSUE TITLE
Importing tensorflow (gpu) as a dependency lowers tracking quality in dlib.correlation_tracker

### ISSUE THREAD

**Issue by mranosa** [davisking/dlib#720]
### Works fine when not imported
```
import cv2
import dlib

# import tensorflow as tf
```
Console log
>('trackingQuality: ', 16.801318305858224)
('trackingQuality: ', 17.87287418733718)
('trackingQuality: ', 18.620422104492167)
('trackingQuality: ', 22.45131422199208)
('trackingQuality: ', 21.623293605567934)
('trackingQuality: ', 17.22383777166709)
>('trackingQuality: ', 15.571112288468427)

### Breaks when imported
```
import cv2
import dlib

import tensorflow as tf
```

Console log
>('trackingQuality: ', 12.8581743346791)
('trackingQuality: ', 9.860590607232695)
('trackingQuality: ', 9.164258417807268)
('trackingQuality: ', 11.591137716989818)
('trackingQuality: ', 8.443019069968415)
('trackingQuality: ', 10.090687246145432)
('trackingQuality: ', 10.356057278570988)
>('trackingQuality: ', 6.5496505695047)

I installed dlib via anaconda `conda install -c menpo dlib` and am on python 2.7 (linux)

**Comment by davisking** [comment]
You probably have some kind of conflict in a BLAS library.  Maybe tf and dlib are compiled against different and incompatible BLAS libraries on your system.

**Comment by mranosa** [comment]
Fixed by upgrading to python 3.5
